tests/utils_test.py

```diff
@@ -0,0 +1,81 @@
+"""
+Unitary tests for utils.py.
+
+:author: xarbulu
+:organization: SUSE Linux GmbH
+:contact: xarbulu@suse.com
+
+:since: 2019-07-05
+"""
+
+# pylint:disable=C0103,C0111,W0212,W0611
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+import logging
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+import pytest
+
+sys.modules['prometheus_client'] = mock.MagicMock()
+
+from hanadb_exporter import utils
+
+
+class TestUtils(object):
+    """
+    Unitary tests for utils.
+    """
+
+
+    def test_format_query_result(self):
+        query_results = mock.Mock()
+        query_results.metadata = [
+            ('column1', 'other_data',), ('column2', 'other_data'), ('column3', 'other_data')]
+        query_results.records = [
+            ('data1', 'data2', 'data3'),
+            ('data4', 'data5', 'data6'),
+            ('data7', 'data8', 'data9')
+        ]
+        formatted_result = utils.format_query_result(query_results)
+
+        assert formatted_result == [
+            {'column1':'data1', 'column2':'data2', 'column3':'data3'},
+            {'column1':'data4', 'column2':'data5', 'column3':'data6'},
+            {'column1':'data7', 'column2':'data8', 'column3':'data9'}
+        ]
+
+    def test_check_hana_range(self):
```

Comment:
we could provide maybe some text with the intention of the test

Reply:
Well, honestly, I think that the name of the test name says enough for the intention of the test. I don't really like to add comments in the test methods if it's not something extremely confusing (i think this is not the case XD)